libvirt.go

```diff
@@ -149,8 +149,17 @@ func (l *Libvirt) Connect() error {
 }
 
 // Disconnect shuts down communication with the libvirt server and closes the
-// underlying net.Conn.
+// underlying net.Conn. Ordering is important here. We want to make sure the
```

Comment:
nit: order comment probably isn't API-level documentation and should probably go inside the fn